src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private ActorDao actorDao = new ActorDaoImpl(sessionFactory);
```

Comment:
You should not create a new instance of ActorDaoImpl here. The ActorDao should be passed as a parameter to the constructor of ActorServiceImpl. This is because we want to follow the Dependency Injection principle. This principle states that a class should not instantiate dependencies by itself, they should be provided from the outside.